spec/controllers/submissions_controller_spec.rb

```diff
@@ -958,6 +962,23 @@
             expect(response).to render_template(partial: 'shared/_poll_job')
           end
 
+          it 'should succeed if it is before the section due date and collect_current is selected' do
+            @assessment_section_properties.update!(due_date: 1.week.from_now)
+            allow(Assignment).to receive_message_chain(
+              :includes, :find
+            ) { @assignment }
+            expect_any_instance_of(SubmissionsController).not_to receive(:flash_now).with(:error, anything)
+            allow(SubmissionsJob).to receive(:perform_later) { Struct.new(:job_id).new('1') }
+
+            post_as @instructor,
+                    :collect_submissions,
+                    params: { course_id: course.id, assignment_id: @assignment.id,
+                              override: true, collect_current: true,
+                              groupings: ([] << @assignment.groupings).flatten }
```

Comment:
this syntax is weird. I know they use it in other tests but is there a reason why `@assignment.groupings.to_a` doesn't work?